app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -116,6 +116,15 @@ <h2>Images</h2>
             </a>
         </p>
 
+        <h2>Hanging list</h2>
+
+        <p class="mt-3">
+            <a class="btn btn-primary"
```

Comment:
I think that we should give the users a warning that this will destroy their current hanging list. Maybe we should only allow/show this if the current reader study is not valid?

Reply:
Good point, I also considered that. I think a warning would probably be sufficient. That way we give the user the freedom to regenerate the list if they would like to do so.